cohort-cli/src/main/java/com/ibm/cohort/cli/MeasureCLI.java

```diff
@@ -79,6 +90,18 @@
 				"--define-return-option" }, description = "Specify define return option for evaluated define statements on measure report. Defaults to NONE.  Valid options are (ALL|BOOLEAN|NONE).")
 		private DefineReturnOptions defineReturnOption = DefineReturnOptions.NONE;
 
+		@Parameter(names = { "--enable-retrieve-cache" }, description = "Enable the use of the retrieve cache.")
+		private boolean enableRetrieveCache = false;
```

Comment:
Do we really want the cache to be disabled by default? Seems like something most people would want by default.

Reply:
I can enable it by default.

Reply:
I changed the options to enable the cache by default via [this commit](https://github.com/Alvearie/quality-measure-and-cohort-service/pull/83/commits/1817b5bbc4f9921ea7b4580dd29a84ed17ac001c).